nevergrad/benchmark/test_experiments.py

```diff
@@ -25,7 +25,7 @@ def test_experiments_registry(name: str, maker: tp.Callable[[], tp.Iterator[expe
     with datasets.mocked_data():  # mock mlda data that should be downloaded
         check_maker(maker)  # this is to extract the function for reuse if other external packages need it
     if name not in {"realworld_oneshot", "mlda", "mldaas", "realworld", "rocket", "mldakmeans",
-                    "naivemltuning", "seqmltuning", "naiveseqmltuning", "mltuning", "control_problem"}:
+                    "naivemltuning", "seqmltuning", "naiveseqmltuning", "mltuning", "control_problem", "naiveseqkerastuning", "seqkerastuning"}:
```

Comment:
better refactor this and exclude all the names with "mltuning" in it.
Then again, why is it not working? Deactivating the test should one be done if it's too slow or requires external data which we don't want in the CI. I don't remember why the first tuning xps did not work.